servant-hmac-auth.cabal

```
@@ -74,12 +74,12 @@ library
                      , http-types ^>= 0.12
                      , http-client >= 0.6.4 && < 0.8
                      , memory >= 0.15 && < 0.19
-                     , mtl ^>= 2.2.2
-                     , servant ^>= 0.18 || ^>= 0.19
-                     , servant-client ^>= 0.18 || ^>= 0.19
-                     , servant-client-core ^>= 0.18 || ^>= 0.19
-                     , servant-server ^>= 0.18 || ^>= 0.19
-                     , transformers ^>= 0.5
+                     , mtl ^>= 2.2.2 || ^>= 2.3
+                     , servant ^>= 0.18 || ^>= 0.19 || ^>= 0.20
+                     , servant-client ^>= 0.19 || ^>= 0.20
+                     , servant-client-core ^>= 0.18 || ^>= 0.19 || ^>= 0.20
```

suggestion: servant-client-core will most likely match servant-client so might as well use the same version bounds?
```suggestion
                     , servant-client-core ^>= 0.19 || ^>= 0.20
```